### DISCUSSION TITLE
Features request [Tesselation]

### DISCUSSION THREAD

**Discussion by e2002e** [UPBGE/upbge#1679] [Ideas/Features for UPBGE 0.3+]
Hi, i post here to ask if you could implement Tesselation, this would make the engine better than armory3d (which is too hard to use for me).

**Comment by lordloki** [comment]
Tesselation is over the opengl 3.3. I think that when the vulkan lands tesselation will be put over the table

**Comment by BluePrintRandom** [comment]
https://developer.blender.org/D13515

I think he means this patch?

**Comment by e2002e** [comment]
In armory3d, the tesselation is on/off in the engine and shaders compute the displace node if any.
Would be great to see that implemented.
